Common/include/CConfig.hpp

```diff
@@ -6373,6 +6572,13 @@ class CConfig {
    */
   su2double GetInlet_Ttotal(string val_index) const;
 
+    /*!
+   * \brief Get the scalar values at an inlet boundary.
+   * \param[in] val_index - Index corresponding to the inlet boundary.
+   * \return The inlet scalar values.
+   */
+  su2double* GetInlet_ScalarVal(string val_index) const;
```

Comment:
This should return const su2double* otherwise it is not really const.

Reply:
yes, fixed it, will be in the next push